ansible/roles/pyassimp/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+# Fix pyassimp issue
+
+- name: Clone pyassimp source code
+  git:
+    repo: 'https://github.com/assimp/assimp.git'
+    version: master
```

Comment:
maybe here it is better to get some exact release. This is because master is always changing and at some point it will break one-liner :)